<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cc</class>
 <widget class="QMainWindow" name="cc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;Ubuntu Mono&quot;;
background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="usu_cc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:   15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="placeholderText">
     <string>Insira o nome de usuário...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="c_senha_cc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:   15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Repita a senha...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="senha_cc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:   15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Insira a senha...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_cc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:   15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Insira o e-mail...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Ubuntu Mono&quot;;</string>
    </property>
    <property name="text">
     <string>CRIAÇÃO DE CONTA</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cc_usu">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Usuário</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="email">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cc_senha">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="csenha">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Confirmar Senha</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bcc">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 62, 90);
alternate-background-color: rgb(0, 62, 90);
border-style: outset;
border-width: 2px;
border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string>CADASTRAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="erro_cc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(237, 212, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>CriptoBot.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>usu_cc</zorder>
   <zorder>c_senha_cc</zorder>
   <zorder>senha_cc</zorder>
   <zorder>email_cc</zorder>
   <zorder>label</zorder>
   <zorder>cc_usu</zorder>
   <zorder>email</zorder>
   <zorder>cc_senha</zorder>
   <zorder>csenha</zorder>
   <zorder>bcc</zorder>
   <zorder>erro_cc</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
